coniferest/aadforest.py

```diff
@@ -135,6 +138,10 @@ def __init__(self,
         self.n_trees = n_trees
         self.tau = tau
         self.C_a = C_a
+
+        if not isinstance(prior_influence, Callable) and not isinstance(prior_influence, float):
```

Comment:
Maybe we should check `prior_influence ` to be a number, not float? It could be annoying if `prior_influence=1` raises an error...

Reply:
Agree